src/main.cpp

```diff
@@ -18,28 +18,28 @@
 #include "VCF_Constants.h"
 #include "VCF_Tasks.h"
 #include "PedalsSystem.h"
-
-
+#include "VCFEthernetInterface.h"
 
 /* Scheduler setup */
 HT_SCHED::Scheduler& scheduler = HT_SCHED::Scheduler::getInstance();
 
-
-
-/* Ethernet message sockets */ // TODO: Move this into its own interface
-qindesign::network::EthernetUDP protobuf_send_socket;
-qindesign::network::EthernetUDP protobuf_recv_socket;
-
-
-
 void setup() {
-    scheduler.setTimingFunction(micros);
-
-    scheduler.schedule(read_adc1_task);
-    scheduler.schedule(read_adc2_task);
-    scheduler.schedule(buzzer_control_task);
+    std::function<unsigned long()> func = micros;
+    scheduler.setTimingFunction(func);
+
+    //scheduler.schedule(read_adc1_task);
+    //scheduler.schedule(read_adc2_task);
+    //scheduler.schedule(buzzer_control_task);
+    // scheduler.schedule(send_vcf_data_task);
+    // scheduler.schedule(recv_vcr_data_task);
+
+    qindesign::network::Ethernet.begin(default_VCF_ip, default_dns, default_gateway, car_subnet);
+    protobuf_send_socket.begin(VCF_SEND_PORT);
+    protobuf_recv_socket.begin(VCF_RECV_PORT);
```

Comment:
if possible, I'd like rename the ports. Each particular message has a designated port, so instead of calling it "send" or "receive", I'd like to define the ports based on their specific messages (i.e. VCRDataPort or VCFDataPort, etc)

Reply:
Also, the things VCF is responsible for are sending/receiving to VCR. This code was just for testing sending to/from laptop, so we should probably remove/replace.